src/Base/QuantityParser.l

```diff
@@ -60,189 +60,188 @@ CGRP     '\,'[0-9][0-9][0-9]
-"G"        yylval = Quantity::Gauss;               return UNIT; // Gauss (1 G = 1e-4 T)
-
-"Wb"       yylval = Quantity::Weber;               return UNIT; // Weber (kg*m^2/s^2/A)
-
-"F"        yylval = Quantity::Farad;               return UNIT; // Farad (s^4*A^2/m^2/kg)
-"mF"       yylval = Quantity::MilliFarad;          return UNIT; // milli Farad
-"\xC2\xB5F"    yylval = Quantity::MicroFarad;      return UNIT; // micro Farad
-"uF"       yylval = Quantity::MicroFarad;          return UNIT; // micro Farad
-"nF"       yylval = Quantity::NanoFarad;           return UNIT; // nano Farad
-"pF"       yylval = Quantity::PicoFarad;           return UNIT; // pico Farad
-
-"H"        yylval = Quantity::Henry;               return UNIT; // Henry (kg*m^2/s^2/A^2)
-"mH"       yylval = Quantity::MilliHenry;          return UNIT; // milli Henry
-"\xC2\xB5H"    yylval = Quantity::MicroHenry;      return UNIT; // micro Henry
-"uH"       yylval = Quantity::MicroHenry;          return UNIT; // micro Henry
-"nH"       yylval = Quantity::NanoHenry;           return UNIT; // nano Henry
-
-"J"        yylval = Quantity::Joule;               return UNIT; // Joule (kg*m^2/s^2)
-"mJ"       yylval = Quantity::MilliJoule;          return UNIT; // milli Joule
-"kJ"       yylval = Quantity::KiloJoule;           return UNIT; // kilo Joule
-"Nm"       yylval = Quantity::NewtonMeter;         return UNIT; // N*m = Joule
-"VAs"      yylval = Quantity::VoltAmpereSecond;    return UNIT; // V*A*s = Joule
-"CV"       yylval = Quantity::WattSecond;          return UNIT; //
-"Ws"       yylval = Quantity::WattSecond;          return UNIT; // W*s = Joule
-"kWh"      yylval = Quantity::KiloWattHour;        return UNIT; // 1 kWh = 3.6e6 J
-"eV"       yylval = Quantity::ElectronVolt;        return UNIT; // 1 eV = 1.602176634e-19 J
-"keV"      yylval = Quantity::KiloElectronVolt;    return UNIT;
-"MeV"      yylval = Quantity::MegaElectronVolt;    return UNIT;
-"cal"      yylval = Quantity::Calorie;             return UNIT; // 1 cal = 4.1868 J
-"kcal"     yylval = Quantity::KiloCalorie;         return UNIT;
-
-"\xC2\xB0" yylval = Quantity::Degree;              return UNIT; // degree         (internal standard angle)
-"deg"      yylval = Quantity::Degree;              return UNIT; // degree         (internal standard angle)
-"rad"      yylval = Quantity::Radian;              return UNIT; // radian
-"gon"      yylval = Quantity::Gon;                 return UNIT; // gon
-"M"        yylval = Quantity::AngMinute;           return UNIT; // minute(Angular)
-"\xE2\x80\xB2" yylval = Quantity::AngMinute;       return UNIT; // minute(Angular)
-"AS"       yylval = Quantity::AngSecond;           return UNIT; // second(Angular)
-"\xE2\x80\xB3" yylval = Quantity::AngSecond;       return UNIT; // second(Angular)
-
-"1"        yylval = Quantity(1.0);                 return ONE;
-{DIGIT}+"."?{DIGIT}*{EXPO}?                         {  yylval = Quantity(num_change(yytext,'.',','));return NUM;  }
```

Comment:
What's the effect to drop the curly braces here? Or is this just cleaning-up the code?

Reply:
The whole commit is just cosmetics. C files are even not regenerated yet as generators leave many whitespace errors in place. Thinking about dropping it entirely for now.

Reply:
Dropped this commit from PR.